src/ImportDefinitionsBundle/Export/Exporter.php

```diff
@@ -103,9 +108,11 @@ public function doExport(DefinitionInterface $definition, $params)
         if ($total > 0) {
             $this->eventDispatcher->dispatch('export_definition.total', new ImportDefinitionEvent($definition, $total));
 
+            $this->artifact = tempnam(PIMCORE_TEMPORARY_DIRECTORY, 'export_definition_artifact');
```

Comment:
Should be configurable to allow to set via CLI params, this is a default / placeholder.